### PR TITLE
SNOW-540761 Support Spark 3.2 Scala 2.13

### PR DESCRIPTION
**Pull request by sfc-gh-mrui**

### CODE REVIEW

**Review by sfc-gh-ema — APPROVED** [review]

File: src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@
 package net.snowflake.spark.snowflake
 
 import java.sql.{Date, Timestamp}
-
+import scala.language.postfixOps
```
Comment (sfc-gh-bli) [inline src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala:21]: what is it?

Comment (sfc-gh-bli) [reply src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala:21]: Is it supported by both Scala 2.12 and 2.13?

**Review by sfc-gh-bli — APPROVED** [review]

File: src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala (hunk 1 above)
Comment (sfc-gh-mrui) [reply src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala:21]: Scala 2.13 needs to import `scala.language.postfixOps` explicitly if postfix operator is used such as `!`.
```
    // clean up download dir
    import sys.process._
    "rm -fr /tmp/test_move_data" !
```

Comment (sfc-gh-mrui) [reply src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala:21]: and yet. It is supported for both.

**Pull request merged**